infra/versions.tf

```diff
@@ -19,7 +19,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.25.0"
+      version = ">= 3.25.0, <= 4.74, != 4.75.0"
```

Comment:
Are `<= 4.74` and `!= 4.75.0` not redundant?

Reply:
They are, but we might eventually want to go higher than 4.75.0, while still keeping that != 4.75.0 as a reminder not to use that broken one.